tests/data/sample_poses.py

```diff
@@ -0,0 +1,90 @@
+"""Sample poses for testing the averaging algorithms.
+
+Authors: Ayush Baid
+"""
+import copy
+from typing import Dict, List, Tuple
+
+import numpy as np
+from gtsam import Pose3, Rot3
+
+
+def generate_relative_from_global(
+    wTi_list: List[Pose3], pose_pairs: List[Tuple[int, int]]
+) -> Dict[Tuple[int, int], Pose3]:
+    """Generate relative poses from global poses.
+
+    Args:
+        wTi_list: global poses.
+        pose_pairs: pairs (i1, i2) to construct relative poses for.
+
+    Returns:
+        Relative pose i2Ti1 for all requested pairs, as a dictionary.
+    """
+    return {(i1, i2): wTi_list[i2].between(wTi_list[i1]) for i1, i2 in pose_pairs}
+
+
+"""4 poses in the circle of radius 5m, all looking at the center of the circle.
+
+For relative poses, each pose has just 2 edges, connecting to the immediate neighbors.
+"""
+CIRCLE_TWO_EDGES_GLOBAL_POSES = [
+    Pose3(Rot3.RzRyRx(0, 0, 0), np.array([0, 0, 0])),
+    Pose3(Rot3.RzRyRx(np.deg2rad(90), 0, 0), np.array([0, 5, 5])),
+    Pose3(Rot3.RzRyRx(np.deg2rad(180), 0, 0), np.array([0, 0, 10])),
+    Pose3(Rot3.RzRyRx(np.deg2rad(270), 0, 0), np.array([0, -5, 5])),
+]
```

Comment:
why are these poses in the yz plane? Maybe simpler in xy plane, and rotate about z? @ayushbaid

Reply:
I am somehow unable to get it set properly. I was initially trying that.

Reply:
What was the issue you were seeing? And which frame were you aiming to define the poses in?

Reply:
I was trying to draw a circle on the x-y plane. And all the cameras point to the center of the circle.

Reply:
PinholeCamera offers a LookAt method, https://github.com/borglab/gtsam/blob/f5ff7aa49f4d6387e3d7a823db140710f343cccf/gtsam/gtsam.i#L1032
But you will need a dummy calibration :( 
Its quite straightforward if its just the x-y plane though.

Reply:
I also cant understand the camera frame here. If z is outwards, I don’t think they look at the center, do they? It does not matter much, but please clarify in the comment.

Reply:
![image](https://user-images.githubusercontent.com/3755532/109061667-f023c900-76b4-11eb-8f9f-7f5a9fc0d7ee.png)
This is what I have. The cameras point towards the center.

Reply:
@ayushbaid do you mind adding a legend with per-axis labels? Maybe we should check that in actually

Reply:
(a label on the matplotlib plot axes and also what red, green , and blue correspond to)

Reply:
Sure. Adding them now.

Reply:
thanks

Reply:
![image](https://user-images.githubusercontent.com/3755532/109065729-fec0af00-76b9-11eb-98d2-81aa25330369.png)
Plot with labels.

Reply:
@johnwlambert looks good?

Reply:
Thanks Ayush. That's much easier to understand now :-)

Can we place the objects on the world xy plane? (which is xz camera for each camera)? Relative measurements will have to be generated in the camera coordinate frame, to be consistent with what DMV produces, right?

Reply:
I directly use an example in GTSAM to get the required configuration.

![image](https://user-images.githubusercontent.com/3755532/109067849-10f01c80-76bd-11eb-86ff-a231913a7b23.png)

Reply:
I see -- the plot you just shared looks good. But doesn't seem the same as the earlier one you posted?

Reply:
Yeah its the cameras point slightly downwards. But it provides us the same thing we wanted to test so I think it is good enough?

Reply:
Oh sure, I didnt realize you were saying we should replace the old circle one with the GTSAM circle example

Reply:
Nice pictures, can we check these into the same directory? For each configuration here?

Reply:
Done